packages/libs/client/README.md

```diff
@@ -535,7 +535,7 @@ We are available via Discord and Github issues:
 [27]:
   https://github.com/kadena-community/kadena.js/blob/main/packages/libs/client-examples/src/example-contract/simple-transfer.ts
 [28]:
-  https://pact-language.readthedocs.io/en/stable/pact-reference.html?highlight=signer#signature-capabilities
```

Comment:
@javadkh2 I got 1 change in your readme.
we are launching the new docs. so this will be the link.
I need an approval from you. if ok

Reply:
Both links are broken!

Reply:
the link is broken for now. it will work when docs is going live!
is that ok for half a day?

I can make it go to: https://alpha-docs.kadena.io/pact/reference/concepts#signature-capabilitiesh289463870
for now and switch it tomorrow?

Reply:
That's fine if it will work eventually.